tests/utils.py

```diff
@@ -71,3 +73,33 @@ def skip_if_destructive_and_sensitive(request, base_url):
 def shorten_tag(tag):
     """Returns the short version of a git tag when given the long (or short) version."""
     return tag.split('-')[0]
+
+
+def get_window_size(self, dimension=None):
+    """Return the current window dimensions."""
+    get_size = self.driver.get_window_size()
+    if not dimension:
+        return get_size
+    if dimension not in get_size:
+        raise IndexError('Unknown dimension: %s' % dimension)
+    return get_size[dimension]
+
+
+def set_window_size(self, width=0, height=0):
+    """Attempt to change the browser window size."""
+    if width >= 1 and height >= 1:
+        self.driver.set_window_size(width, height)
+        sleep(1.0)
+    # return self.get_window_size()
```

Comment:
Do not merge code with commented code. If there is commented code it either needs to be deleted or be necessary.